src/openrct2/world/Climate.cpp

```diff
@@ -43,7 +44,7 @@ struct WeatherTransition
 };
 
 extern const WeatherTransition* ClimateTransitions[4];
-extern const WeatherState ClimateWeatherData[6];
+extern const WeatherState ClimateWeatherData[9];
```

Comment:
Could you introduce a `constexpr` or use `std::size` here, instead of a hardcoded number?